Amazon.Ion.ObjectMapper/Attributes.cs

```diff
@@ -36,6 +36,26 @@ public IonPropertyName(string name)
         public string Name { get; }
     }
 
+    public class IonPropertyGetter : Attribute
```

Comment:
Should we start adding comments for these? Can you chat w/ @alpian about this?

Reply:
Do we want the comments added in this PR or a later PR which will handle all the Attribute comments?

Reply:
@justing-bq - was there a decision around this? If not, then please add them as we go.